@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -628,6 +634,9 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                         if (computeTexCoords !== undefined) {
                             const { u, v } = computeTexCoords(pos);
                             ringContour.push(u, v);
+                            if (ringPoints !== undefined) {
+                                ringPoints.push(u, v);
+                            }
```

Comment:
Rings, contours, ringPoints and stride are a bit messy. In a follow up change I will try to rework the way we work with multi-polygons.

Reply:
Agree, we need to clean this one up.